### ISSUE TITLE
Issues setting path after update

### ISSUE THREAD

**Issue by andoruB** [rimpy-custom/RimPy#30]
OS: Manjaro x64 Cinnamon - 5.15.2-2
RimWorld 1.3.3200
RimPy 1.2.4.21

Switched over on to the newest version of RimPy and been having issues updating mods. Getting I/O errors for some reason.  
Tried to clean the config folder for RimPy (~/.config/RinPy) and SteamCMD and start from scratch.  
Unable now to set the folders for the game and the mods/SteamCMD, when I press the browse button, program ends and I get this:   
  
```
$ ./RimPy.sh
Gtk-Message: 23:27:41.284: Failed to load module "xapp-gtk3-module"
Gtk-Message: 23:27:41.284: Failed to load module "canberra-gtk-module"
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
Qt WebEngine resources not found at //resources. Trying parent directory...
Qt WebEngine resources not found at /. Trying application directory...
Installed Qt WebEngine locales directory not found at location /translations/qtwebengine_locales. Trying application directory...
Qt WebEngine resources not found at //resources. Trying parent directory...
Qt WebEngine resources not found at /. Trying application directory...
Installed Qt WebEngine locales directory not found at location /translations/qtwebengine_locales. Trying application directory...
Qt WebEngine resources not found at //resources. Trying parent directory...
Qt WebEngine resources not found at /. Trying application directory...
Installed Qt WebEngine locales directory not found at location /translations/qtwebengine_locales. Trying application directory...
Qt WebEngine resources not found at //resources. Trying parent directory...
Qt WebEngine resources not found at /. Trying application directory...
Installed Qt WebEngine locales directory not found at location /translations/qtwebengine_locales. Trying application directory...
js: Uncaught ReferenceError: qt is not defined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'parentElement' of null
3.4767655130126514

(RimPy:88703): Gtk-WARNING **: 23:27:50.734: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognised image file format (gdk-pixbuf-error-quark, 3)
Bail out! Gtk:ERROR:../../../../gtk/gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /org/gtk/libgtk/icons/16x16/status/image-missing.png: Unrecognised image file format (gdk-pixbuf-error-quark, 3)
./RimPy.sh: line 2: 88703 Aborted                 (core dumped) LD_PRELOAD="libpcre.so.3:$LD_PRELOAD" ./RimPy
```

Tried to search for the modules mentioned above, but no such things are available on the Arch repos.  
There's also no org folder on the root, so I'm not sure if that's something gone awry, or an issue with Arch and distros under it are not supported?  
Previously to clearing the settings, I was able to click on the browse, but once selecting the path, the app would crash. Unfortunately I wasn't able to see the reason for this as I didn't have it running in a terminal, so no logs for that.

**Comment by andoruB** [comment]
Any progress on this issue...?

**Comment by rimpy-custom** [comment]
May you test with new RimPy pre-release?

**Comment by NBZion** [comment]
this is a duplicate of #64

**Comment by andoruB** [comment]
@NBZion My dude, calm your tits, this issue was submitted long before #64 was ever a thing.

@rimpy-custom I'm afraid the issue still stands. Same exact crash, same exact error messages.

**Comment by jvzr** [comment]
Similar issue on Arch 5.18.16 with Gnome 42: clicking on the buttons to select the paths (which weren't auto-detected as I too store the game on a secondary disk) crash the app. Note that I'm using Steam, and the Steam Workshop mods.

Here's the launch log, as Logs/rimpy.log seems devoid of useful information related to this crash: 

```
🐟 common/RimWorld/RimPy_Linux ⟩ ./RimPy.sh
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
Ошибка открытия URL: https://api.github.com/repos/rimpy-custom/RimPy/releases
Sandboxing disabled by user.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
ERROR: ld.so: object 'libpcre.so.3' from LD_PRELOAD cannot be preloaded (cannot open shared object file): ignored.
js: Uncaught ReferenceError: qt is not defined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'lastChild' of undefined
js: Failed to report error: ReportError() was called without an error to report.
js: Uncaught TypeError: Cannot read property 'parentElement' of null
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version
5.20877823800015
Ошибка открытия URL: https://content-system.gog.com/products/1094900565/os/windows/builds?generation=2
Ошибка открытия URL: https://rimworldwiki.com/wiki/Version/Current_version

(RimPy:5185): Gtk-WARNING **: 08:13:58.231: Could not load a pixbuf from icon theme.
This may indicate that pixbuf loaders or the mime database could not be found.
**
Gtk:ERROR:gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Vimix-dark/16/actions/image-missing.svg: Unable to load image-loading module: /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib/librsvg-2.so.2: undefined symbol: pango_attr_overline_color_new (gdk-pixbuf-error-quark, 5)
Bail out! Gtk:ERROR:gtkiconhelper.c:494:ensure_surface_for_gicon: assertion failed (error == NULL): Failed to load /usr/share/icons/Vimix-dark/16/actions/image-missing.svg: Unable to load image-loading module: /usr/lib/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-svg.so: /usr/lib/librsvg-2.so.2: undefined symbol: pango_attr_overline_color_new (gdk-pixbuf-error-quark, 5)
/mnt/games/jvzr/Steam Library/steamapps/common/RimWorld/RimPy_Linux/RimPy.sh: line 2:  5185 Aborted                 (core dumped) LD_PRELOAD="libpcre.so.3:$LD_PRELOAD" ./RimPy
```

**Comment by rimpy-custom** [comment]
Closing for now. Reopen if issue wasn't fixed with latest Linux build 1.2.6.28.
